config/templates/datacite.template.yml

```diff
@@ -1,6 +1,10 @@
 default: &default
+  rest_api: 'https://api.test.datacite.org'
   user: apitest
   password: apitest
+  prefix: '10.33555'
+  log_api: false
+  log_level: warn
   shoulder:
     ark: 'ark:99999/fk4'
     doi: 'doi:10.5072/FK2'
```

Comment:
In the collapsed section below I still see some ezid references in the file.  Can these be removed?

Reply:
Just to clarify, it's not actually the highlighted lines that you see with this comment.  It's the lines below those (e.g. `ezid_test_user: apitest`).

Reply:
Not anymore!